src/utils/constants.js

```diff
@@ -1,2 +1,9 @@
 // eslint-disable-next-line import/prefer-default-export
 export const DEFAULT_PAGE_LIMIT = 8;
+
+export const transactionStepsNames = {
+  step1: 'add-parties',
+  step2: 'assign-tasks',
+  step3: 'upload-documents',
+  step4: 'close-deal',
```

Comment:
```suggestion
  addParties: 'add-parties',
  assignTasks: 'assign-tasks',
  UploadDocuments: 'upload-documents',
  closeDeal: 'close-deal',
```
You can write docs for the order of the steps